### ISSUE TITLE
a file query of . crash the complet nodeRED-Server

### ISSUE THREAD

**Issue by AleksCee** [Supergiovane/node-red-contrib-tts-ultimate#9]
Hi.

if I build an URL like this: http://noderedserver:1980/tts?f=. the nodeRed will completely crash:

```
25 Jan 18:26:23 - [red] Uncaught Exception:
25 Jan 18:26:23 - ReferenceError: error is not defined
    at fine (/data/node_modules/node-red-contrib-tts-ultimate/ttsultimate/ttsultimate-config.js:475:107)
    at ReadStream.<anonymous> (/data/node_modules/node-red-contrib-tts-ultimate/ttsultimate/ttsultimate-config.js:461:25)
    at ReadStream.emit (events.js:198:13)
```

Alex

**Comment by Supergiovane** [comment]
Thank you Alex
I’ll fix it immediately.

**Comment by Supergiovane** [comment]
Fixed in 1.0.10 out in minutes.

**Comment by AleksCee** [comment]
@Supergiovane on facebook I talk about an other bug. I now have send it as Mail because I don’t want to make it public before you can fix it.
